go.sum

```diff
@@ -22,12 +22,15 @@ cloud.google.com/go/bigquery v1.3.0/go.mod h1:PjpwJnslEMmckchkHFfq+HTD2DmtT67aNF
 cloud.google.com/go/bigquery v1.4.0/go.mod h1:S8dzgnTigyfTmLBfrtrhyYhwRxG72rYxvftPBK2Dvzc=
 cloud.google.com/go/bigquery v1.5.0/go.mod h1:snEHRnqQbz117VIFhE8bmtwIDY80NLUZUMb4Nv6dBIg=
 cloud.google.com/go/bigquery v1.7.0/go.mod h1://okPTzCYNXSlb24MZs83e2Do+h+VXtc4gLoIoXIAPc=
+cloud.google.com/go/bigquery v1.8.0 h1:PQcPefKFdaIzjQFbiyOgAqyx8q5djaE7x9Sqe712DPA=
```

Comment:
Is there a specific reason you need to include this? I know it was added when Aviv first opened the PR. Unless you've explicitly added or updated go modules, it's probably not necessary - see if you can remove it.

Reply:
I didn't mean to add any dep explicitly when I wrote the poc - I just run `make'.

Reply:
@sidsankhe you can remove go.sum from the git index for this PR, it's probably not necessary